src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,56 @@
  */
 
 public class Plane {
+    private String planeModel;
+    private String planeEngineType;
+    private int planeSize;
+    private int planeSpeed;
+    private String planeColor;
 
+    private Plane(PlaneBuilder planeBuilder) {
+        this.planeModel = planeBuilder.planeModel;
```

Comment:
why do you set only one field?

Reply:
Fixed

Reply:
где?

Reply:
Everything is there, I checked it on QA as well, I re-pushed it just in case, right now you should see that this constructor is filled with all required data :)